src/model/inline-document.ts

```diff
@@ -39,7 +39,7 @@ export interface InlineDocInfo<AstNode> {
  * @template N The AST node type
  */
 export class ScannedInlineDocument implements ScannedFeature, Resolvable {
-  type: 'html'|'javascript'|'css'|/* etc */ string;
+  type: 'html'|'js'|'css'|/* etc */ string;
```

Comment:
this might be a breaking change?

Reply:
This 'javascript' type value was never actually used anywhere and was just an incorrect declaration insofar as I could tell.  I had to trace code around to verify this, but there were no cases I could find where the type value 'javascript' was used but there actually was one where 'js' was being used and so was against the type def anyways.  So from that perspective, I would expect this not to be a breaking change since the Analyzer never generated an inline document with that value for type.

Reply:
To put this another way: this is not a breaking change.  It's a bug fix; the type was wrong here AFAICT.

Reply:
@rictic ^ would you agree?

Reply:
Yep. This also won't cause compile errors, because the fallback type `string` is still valid.